tools/cloud-build/daily-tests/tests/hcls-v6.yml

```diff
@@ -23,7 +23,7 @@ zone: us-central1-c
 workspace: /workspace
 blueprint_yaml: "{{ workspace }}/examples/hcls-blueprint-v6.yaml"
 network: "{{ test_name }}-net"
-login_node: "{{ slurm_cluster_name }}-login-*"
+login_node: "{{ slurm_cluster_name }}-slurm-login-*"
```

Comment:
I am confused, how did the name get longer? I thought we were removing a "login" from the name.

Reply:
NVM, I see.